### ISSUE TITLE
Window resize after init

### ISSUE THREAD

**Issue by seven1240** [floooh/sokol#442]
is there a way to resize a window after created dynamically?  e.g. `win.resize(width, height);`

thanks.

**Comment by floooh** [comment]
No, but it would make sense (at least on platforms that support it).
